dbms/src/test/java/org/polypheny/db/constraints/UniqueConstraintTest.java

```diff
@@ -518,14 +516,16 @@ public void batchUpdateTest() throws SQLException {
     }
 
 
-    @Test
-    public void updateNoConflictTest() throws SQLException {
+    @ParameterizedTest(name = "{index}. Create Index: {0}")
+    @ValueSource(booleans = { false, true })
+    @Tag("cottontailExcluded")
```

Comment:
Why is this cottontail exclude necessary?

Reply:
There seem to be weird behavior with referential updates, similar to https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1520963686, I would also handle it similarly.